src/viur/core/bones/relational.py

```diff
@@ -405,74 +405,53 @@ def serialize(self, skel: "SkeletonInstance", name: str, parentIndexed: bool) ->
 
         :raises AssertionError: If a programming error is detected.
         """
+
+        def serialize_dest_rel(in_value: dict | None = None):
```

Comment:
a type hint here would of course be the cherry on the cake. But you can't have everything ...

Reply:
```suggestion
        def serialize_dest_rel(in_value: dict | None = None) -> (dict | None, dict | None):
```